doc/sphinx/source/releases.rst

```diff
@@ -19,11 +19,15 @@ The code is tagged to contextualize the versioning, mark significant
 developments and to mark versions used to produce main results. The
 significant releases since the code was made public are:
 
+`Version 4.0.10 <https://github.com/NNPDF/nnpdf/releases/tag/4.0.10>`_
+    Last release fully compatible with the 4.0 era runcards.
```

Comment:
I don't know if you want to change the 4.0.9 description concerning "Last" - or we could do "Final-Final" :innocent:

Reply:
The 4.0.9 is compatible with the entire pipeline, 4.0.10 only with the runcards. As in, the runcards work but now they use eko instead of apfel, etc.

Clearly it was not clear, I'll reword it a bit.